internal/services/compute/managed_disk_resource.go

```diff
@@ -976,3 +978,19 @@ func resourceManagedDiskDelete(d *pluginsdk.ResourceData, meta interface{}) erro
 
 	return nil
 }
+
+func shutDownOnResize(disk compute.Disk, oldSizeGB, newSizeGB int) bool {
+	// OS disks can't be expanded without downtime.
+	if disk.OsType != "" {
+		return true
+	}
+	// Standard HDD disks can't be expanded without downtime
+	if disk.Sku.Name == compute.DiskStorageAccountTypesStandardLRS {
```

Comment:
Ultra SSD is still in preview as well, let's also exclude `compute.DiskStorageAccountTypesUltraSSDLRS`?

Reply:
@myc2h6o done!